cli/wallet.go

```diff
@@ -39,6 +39,8 @@ var walletCmd = &cli.Command{
 		walletVerify,
 		walletDelete,
 		walletMarket,
+		WalletIDSp,
+		WalletIDAddr,
 	},
```

Comment:
The rest of the functions in this list are not public ( lower-case `w` ). Better align.